common/cont-lib/usr/share/cont-docs/70-general.txt

```diff
@@ -0,0 +1,16 @@
+General container help
+----------------------
+
+Run `docker run THIS_IMAGE container-usage` to get this help.
```

Comment:
To not forget, if you add something like
/usr/share/cont-docs/50-openshift-httpd.txt in your additional layer, you'll
get your help text into 'container-usage' automatically.